coral/src/app/layout/main-navigation/MainNavigation.test.tsx

```diff
@@ -0,0 +1,226 @@
+import MainNavigation from "src/app/layout/main-navigation/MainNavigation";
+import { cleanup, screen, render, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+// mock out svgs to avoid clutter
```

Comment:
What kind of clutter are we talking about?

Reply:
The whole `<svg>` is printed when doing `screen.debug()` or in case of an error. It can get really long, so you can't even see all DOM output. 

I tried to add a more general mock for that, but have to look into it deeper: 
- We could do it in `setupAfterEnv`, but that would require us to import React there to enable JSX and I don't want that in every testsuite. Without React, I can't pass the right element with test-id to get rendered in place. We _could_ render a simple text 🤔  and look for that. But that would be my last resort. 
- I didn't get a helper function to work, because `jest.mock` has to be defined on top -.- so it does not really work with a helper function. 

I would add a follow up ticket for that.